pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 testpaths = tests/ morango/
-DJANGO_SETTINGS_MODULE = morango.settings
+DJANGO_SETTINGS_MODULE = testapp.settings
```

Comment:
right they should work in the testapp context. But I am writing all my tests to happen in the testapp.